src/verilog/verilog_elaborate.cpp

```diff
@@ -273,11 +273,38 @@ void verilog_typecheckt::collect_symbols(const verilog_declt &decl)
       symbol.name = hierarchical_identifier(symbol.base_name);
       symbol.pretty_name = strip_verilog_prefix(symbol.name);
 
+      genvars[symbol.base_name] = -1;
+
       add_symbol(symbol);
     }
   }
 }
 
+void verilog_typecheckt::collect_symbols(
+  const verilog_module_itemt &inst,
+  const verilog_instt::instancet &instance)
+{
+  /*
+  bool primitive = inst.id() == ID_inst_builtin;
+  irep_idt instantiated_module_identifier =
+    verilog_module_symbol(id2string(inst.get(ID_module)));
+
+  // add symbol for the module instance
+  symbolt symbol;
+
+  symbol.mode = mode;
+  symbol.base_name = instance.base_name();
+  symbol.type =
+    typet(primitive ? ID_primitive_module_instance : ID_module_instance);
+  symbol.module = module_identifier;
+  symbol.name = hierarchical_identifier(symbol.base_name);
+  symbol.pretty_name = strip_verilog_prefix(symbol.name);
+  symbol.value.set(ID_module, instantiated_module_identifier);
+
+  add_symbol(std::move(symbol));
+*/
```

Comment:
It would be helpful to have a comment in the source code as to why all this code is commented out.

Reply:
I'll clean up.